source/javascripts/components/unified-editor/StepSelectorDrawer/components/SelectableStepBundleCard.tsx

```diff
@@ -31,6 +31,7 @@ const SelectableStepBundleCard = (props: SelectableStepBundleCardProps) => {
           textAlign="left"
           _hover={{ borderColor: 'border/hover' }}
           onClick={handleClick}
+          marginBottom="12"
```

Comment:
Please avoid this inside a component. Use flex for the parent list, or add margin from a prop.
I know it works here, because we use this component only one place. But its a bad pattern - if we need this component somewhere else this margin will be disturbing.